v22.1/take-and-restore-encrypted-backups.md

```diff
@@ -7,62 +7,159 @@ docs_area: manage
 
-### AWS KMS URI format
+### URI formats
+
+#### AWS KMS URI format
 
 The AWS KMS URI must use the following format:
 
 ~~~
-aws:///<cmk>?AUTH=<auth_type>&REGION=<region>
+aws:///{key}?AUTH={auth_type>}&REGION={region}
 ~~~
 
 The AWS URI **requires** the following:
 
  Component                  | Description
 ----------------------------+------------------------------------------------------------------------
 `aws:///`                   | The AWS scheme. Note the triple slash (`///`).
-`<cmk>`                     | The key identifiers used to reference the CMK that should be used to encrypt or decrypt. For information about the supported formats, see the [AWS KMS docs](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id).
-`AUTH=<auth_type>`          | The user-specified credentials.<ul><li>If the `AUTH` parameter is not provided, the AWS connections default to `specified` and the access keys must be provided in the URI parameters (e.g., `AWS_ACCESS_KEY_ID=<key_id>&AWS_SECRET_ACCESS_KEY=<secret_key>`).</li><li>If `AUTH=implicit`, the access keys can be omitted and the [credentials will be loaded from the environment](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/).</li></ul>
-`REGION=<region>`           | The region of the CMK.
+`{key}`                     | The key identifiers used to reference the KMS key that should be used to encrypt or decrypt. For information about the supported formats, see the [AWS KMS docs](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id).
+`AUTH=<auth_type>`          | The user-specified credentials. If you use `AUTH=specified`, you must provide access keys in the URI parameters (e.g., `AWS_ACCESS_KEY_ID=<key_id>&AWS_SECRET_ACCESS_KEY=<secret_key>`). If you use `AUTH=implicit`, the access keys can be omitted and the [credentials will be loaded from the environment](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/). For details on setting up and using the different authentication types, see [Authentication](use-cloud-storage-for-bulk-operations.html#authentication).
+`REGION=<region>`           | The region of the KMS key.
+
+See AWS's [KMS keys](https://docs.aws.amazon.com/kms/latest/developerguide/create-keys.html) documentation for guidance on creating an AWS KMS key.
+
+#### Google Cloud KMS URI format
+
+The Google Cloud KMS URI must use the following format:
+
+~~~
+gs:///projects/{project name}/locations/{location}/keyRings/{key ring name}/cryptoKeys/{key name}?AUTH={auth_type}{&CREDENTIALS=}
```

Comment:
Typo here with the {&CREDENTIALS=} part?

Reply:
Yes good catch, thanks!